StyleCop.Analyzers/StyleCop.Analyzers/MaintainabilityRules/SA1413UseTrailingCommasInMultiLineInitializers.cs

```diff
@@ -73,6 +74,23 @@ public override void Initialize(AnalysisContext context)
 
             context.RegisterSyntaxNodeAction(HandleObjectInitializerAction, ObjectInitializerKinds);
             context.RegisterSyntaxNodeAction(HandleAnonymousObjectInitializerAction, SyntaxKind.AnonymousObjectCreationExpression);
+            context.RegisterSyntaxNodeAction(HandleEnumMemberDeclarationAction, SyntaxKind.EnumDeclaration);
+        }
+
+        private static void HandleEnumMemberDeclarationAction(SyntaxNodeAnalysisContext context)
+        {
+            var initializer = (EnumDeclarationSyntax)context.Node;
+            var lastMember = initializer.Members.LastOrDefault();
+            if (lastMember == null)
+            {
+                return;
+            }
+
+            var commas = initializer.ChildNodesAndTokens().OfType<SyntaxNodeOrToken>().Where(token => token.IsKind(SyntaxKind.CommaToken));
+            if (initializer.Members.Count() != commas.Count())
+            {
+                context.ReportDiagnostic(Diagnostic.Create(Descriptor, lastMember.GetLocation()));
+            }
```

Comment:
The code above should follow the same pattern as the methods below. Specifically you can check `EnumDeclarationSyntax.Members.Count` against `EnumDeclarationSyntax.Members.SeparatorCount`. That will work just as well and it will perform quite a bit better.